src/analytics.cc

```diff
@@ -46,6 +46,14 @@ void Analytics::TrackOs(const std::string client_id,
     Track(client_id, "os", ss.str());
 }
 
+void Analytics::TrackWindowSize(const std::string client_id,
+                                const toggl::Rectangle rect) {
+    std::stringstream ss;
+    ss << "winsize-"<< rect.str();
```

Comment:
We should change winsize to `mainsize` as we are going to add tracking for other windows of the app in the future.

Also we want to track platform so the final format should be `os/mainsize`

`os` can be `win`, `mac` or `linux`